app/components/SimulationViewer/SimulationViewer.tsx

```diff
@@ -35,35 +54,107 @@ export default function SimulationViewer({ jsonFile, chefsFormId }: SimulationVi
   useEffect(() => {
     // reset context/results when a new submission is selected
     resetContextAndResults();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Disabling seems to be recommended against by a few different sources, so I'm not sure what the best course of action would be.

Reply:
True true. Leave it how it is now. Looking at it again, I do think the underlying issue with some of these could be updated so that the error doesn't show up. But probably not worth it for now.